internal/os_util.py

```diff
@@ -128,3 +129,12 @@ def get_file_version(filename):
     except:
         logging.exception('Error getting file version for %s', filename)
     return version
+
+def pc_name():
+    """ Grabs the hostname and Local IP address of the machine and returns hostname-IP """
+    try:
+        with socket.socket(socket.AF_INET, socket.SOCK_DGRAM) as s:
+            s.connect(("8.8.8.8", 80))
+            return socket.gethostname() + "-" + s.getsockname()[0]
+    except Exception as e:
+        logging.error("Error getting pc_name: ", e)
```

Comment:
Almost there.  This should probably return platform.uname()[1] outside of the exception as a default so that self.pc_name always has a value like it did before.

Reply:
Got it thanks! if would like, please take a look at the other PR's.